From d8cb2b998329adbae216f6836f3affa3ce21588e Mon Sep 17 00:00:00 2001
From: Philip Chimento <philip.chimento@gmail.com>
Date: Tue, 30 Apr 2019 13:30:27 -0500
Subject: [PATCH 2/7] build: Fix library install name on macOS

In order for the library to work under JHbuild, it has to have an install
name with an absolute path, since it will be installed in a shared
location.

https://bugzilla.mozilla.org/show_bug.cgi?id=1379536
---
 config/rules.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/rules.mk b/config/rules.mk
index 2eb9e61d..79f02dc3 100644
--- a/config/rules.mk
+++ b/config/rules.mk
@@ -334,7 +334,7 @@ ifdef SHARED_LIBRARY
 ifdef MOZ_IOS
 _LOADER_PATH := @rpath
 else
-_LOADER_PATH := @executable_path
+_LOADER_PATH := $(abspath $(prefix))/lib
 endif
 EXTRA_DSO_LDOPTS	+= -dynamiclib -install_name $(_LOADER_PATH)/$(SHARED_LIBRARY) -compatibility_version 1 -current_version 1 -single_module
 endif
-- 
2.21.0

